db/migrate/20240606191213_create_follow_relationships.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFollowRelationships < ActiveRecord::Migration[7.1]
+  def change
+    create_table :follow_relationships do |t|
+      t.integer :follower_id
+      t.integer :followed_id
+
+      t.timestamps
+    end
+    add_index :follow_relationships, :follower_id
+    add_index :follow_relationships, :followed_id
+    add_index :follow_relationships, [:follower_id, :followed_id], unique: true
```

Comment:
Let's use symbols arrays whenever we can!
```suggestion
    add_index :follow_relationships, %i[follower_id followed_id], unique: true
```